app/manifest/v2/_base.json

```diff
@@ -20,10 +20,10 @@
   "commands": {
     "_execute_browser_action": {
       "suggested_key": {
-        "windows": "Alt+Shift+M",
-        "mac": "Alt+Shift+M",
-        "chromeos": "Alt+Shift+M",
-        "linux": "Alt+Shift+M"
+        "windows": "Ctrl+Shift+M",
+        "mac": "Ctrl+Shift+M",
```

Comment:
```suggestion
        "mac": "Command+Shift+M",
```

Reply:
Echoing what @ulissesferreira said, Chrome on Mac shows user profiles when I do this

Reply:
Chrome on Windows **also** shows user profiles when I do this.  We can't make this change.